src/time-map.scm

```diff
@@ -5,6 +5,7 @@
 (use-modules (opencog python))
 
 (StateLink (ConceptNode "last person who spoke") (NumberNode "0"))
+(StateLink (ConceptNode "previous person who spoke") (NumberNode "0"))
```

Comment:
what is the difference between "last" and "previous"? Is "previous" the same thing as "second from last"? adding a few remarks to clarify this would be nice.